src/lib/containers/widgets/query-filter/EnumFacetFilter.tsx

```diff
@@ -43,7 +44,8 @@ function valueToLabel(
   if (eh) {
     displayValue = eh ? eh.name : `unknown (${value})`
   }
-  return `${displayValue} (${count})`
+
+  return `${displayValue}`
```

Comment:
no more count?

Reply:
not here. We get it straight in html 
https://github.com/Sage-Bionetworks/Synapse-React-Client/pull/422/files/08608266ddb63fe526422a151fb198777bf06d9e#diff-733576c09bfbc11b32f3d4c6a64d28e0R137